### PR TITLE
fix: duplication in descriptions HTML foundations

### LINKED ISSUE #52821: [Odin project] HTML Foundations Questions A and B have duplicate text
### Describe the Issue

The first two challenges in the HTML foundations have the exact same text.

https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-a
https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-b

I feel like it would be better to update the text for one of these challenges so campers don't read through the same thing twice.

The text is mainly talking about the structure and syntax of an HTML element.
So it feels like that text better matches the question in lesson B. 

So we could probably just update the text for lesson A.

### Affected Page

https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-a

### Your code

n/a

See explanation above

### Expected behavior

We shouldn't have duplicated text instructions for two lessons in a row

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### LINKED ISSUE #52824: [Odin project] HTML Foundations Questions H, I and J have duplicate text
### Describe the Issue

These three challenges in the HTML foundations have the exact same text.

https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-h
https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-i
https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-j

We should update the text for all three lessons so the campers doesn't have to the read the same text again.

happy to hear suggestions 😃 

### Affected Page

https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-h

https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-i

https://www.freecodecamp.org/learn/the-odin-project/top-learn-html-foundations/html-foundations-question-j

### Your code

N/A

see explanation 

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Sembauke**
closes #52824
closes #52821

### CODE REVIEW

**Review by jdwilkin4 — APPROVED** [review]
tested it locally and the changes look good to me 👍

File: curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-e.md [hunk 1]
```diff
@@ -1,37 +1,63 @@
 ---
-id: 6374f208de18c50e48ba767b
+id: 637f4e2f72c65bc8e73dfe22
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-e.md:2]: This ID changed. I'll need to sync it to the i18n instances.

File: curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-f.md [hunk 2]
```diff
@@ -1,65 +1,47 @@
 ---
-id: 637f4e2f72c65bc8e73dfe22
+id: 637f4e3672c65bc8e73dfe23
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-f.md:2]: This one too.

File: curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-g.md [hunk 3]
```diff
@@ -1,47 +1,33 @@
 ---
-id: 637f4e3672c65bc8e73dfe23
-videoId: V8UAEoOvqFg
+id: 6376327e2724a688c04636e3
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-g.md:2]: And this.

File: curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-h.md [hunk 4]
```diff
@@ -1,36 +1,32 @@
 ---
-id: 6376327e2724a688c04636e3
+id: 637633162724a688c04636e4
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-h.md:2]: .

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by naomi-lgbt** [comment]
Once #52964 is in, I'll rebase and run a sync.

**Comment by Sembauke** [comment]
thanks Naomi.

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**